src/components/FormModal/FormModal.js

```diff
@@ -149,13 +149,14 @@ export default function FormModal({
    * as well as the state of the day and timeOfDay to when the product was clicked (e.g. monday morning)
    * @type {function}
    * @param {string} name
+   * @param {string} timeOfDay
    */
 
-  function handleMorningClicked(name) {
+  function handleTimeOfDayClicked(name, timeOfDay) {
     const conflictingId = findConflictingProductId(
       name,
       routineData,
-      "morning",
+      timeOfDay,
```

Comment:
i mean, morning is more specific but sure

Reply:
yeah, but now it can be also evening, ya know?